kiwmi/desktop/desktop.c

```diff
@@ -43,6 +65,39 @@ desktop_init(struct kiwmi_desktop *desktop, struct wlr_renderer *renderer)
     wlr_xdg_output_manager_v1_create(
         server->wl_display, desktop->output_layout);
 
+    desktop->scene = wlr_scene_create();
+    if (!desktop->scene) {
+        wlr_log(WLR_ERROR, "failed to create scene");
+        return false;
+    }
+
+    // different ordering:
+    // - background, bottom, views, top
+    // - bg+bot+views+top popups (possibly unified)
+    // - overlay, overlay popups
```

Comment:
Any comments on this pls? I believe it’s the ‘correct’ order, but am unsure if the four popup strata should be reduced to a single stratum instead.

```diff
- current state
+ proposal
  --------
  background
  bottom
  views
  top
- background popups
- bottom popups
- view popups
- top popups
+ popups (except overlay)
  overlay
  overlay popups
```

Reply:
Hmm. I think i’ll keep them separated, as that is only a lil bit more complicated to initialize, but then easy to work with. Unless, of course, it becomes a single node with overlay’s popups.